### DISCUSSION TITLE
Scenarios execution order

### DISCUSSION THREAD

**Discussion by lufton** [getgauge/taiko#2616] [General]
Can somebody explain how to control scenarios execution order within spec file?
I have case when scenarios that are at the bottom of the spec file are running before those that are on top. And that's confusing as I want to be able to control execution order as I have dependent scenarios in one spec file.

**Comment by zabil** [comment]
There is info about that here
https://docs.gauge.org/faqs.html?#execution-1

At the moment execution order cannot be controlled and it's recommended that the tests be written in a way that they are not dependent of other scenarios.

There's a discussion about it here too
https://github.com/getgauge/gauge/issues/1342
